app/components/widgets/rain-graph-directive.js

```diff
@@ -12,25 +12,46 @@ var rainGraphWidget = function (ApiService) {
   * @param  {object} attrs - contents of the attributes
   * @return {object}       - Object with angular config
   */
+  var SIZE = 1200;
   var rainGraphWidgetLink = function (scope, elem) {
+    var months = ['jan', 'feb', 'mrt', 'apr', 'mei', 'jun', 'jul', 'aug',
+      'sep', 'okt', 'nov', 'dec'];
     var chart = nv.models.multiBarChart()
-      .x(function (d) { return Date.parse(d[0]); })
-      .y(function (d) {
-        console.log(d)
-        return d[1];
-      })
-      .reduceXTicks(true)   // If 'false', every single x-axis tick label will be rendered.
-      .rotateLabels(0)      // Angle to rotate x-axis labels.
-      .showControls(true)   // Allow user to switch between 'Grouped' and 'Stacked' mode.
+      .x(function (d) { return months[new Date(d[0] + 1296000000).getUTCMonth()]; })
```

Comment:
Akkoord, Dat is ca. 15 dagen en zorgt ervoor dat de juiste maand in tekst wordt getoond.